IS7024/Pages/LudlowGarage.cshtml.cs

```diff
@@ -18,26 +19,31 @@ public void OnGet()
             using (var webClient = new WebClient())
             {
                 string sunJSON = webClient.DownloadString("https://api.sunrise-sunset.org/json?lat=39.7970&lng=83.8255&date=2020-11-02");
-                var sun = QuickTypeSun.Sun.FromJson(sunJSON);
-                var results = sun.Results.Sunset;
-                string s = results;
-                string[] vars = s.Split(':');
-                var newtime = Convert.ToInt16(vars[0]);
-                int realtime = newtime + 6 - 12;
-                string itstime = "~" + realtime + ":" + vars[1] + " PM";
-                ViewData["sun"] = itstime;
-                
+                JSchema sunSchema = JSchema.Parse(System.IO.File.ReadAllText("SunSchema.json"));
+                JObject sunObject = JObject.Parse(sunJSON);
+                if (sunObject.IsValid(sunSchema))
+                {
+                    QuickTypeSun.Sun sun = QuickTypeSun.Sun.FromJson(sunJSON);
+                    string s = sun.Results.Sunset;
+                    string[] vars = s.Split(':');
+                    var newtime = Convert.ToInt16(vars[0]);
+                    int realtime = newtime + 6 - 12;
+                    string itstime = "~" + realtime + ":" + vars[1] + " PM";
+                    ViewData["sun"] = itstime;
+                }
 
                 string jsonString = webClient.DownloadString("https://api.songkick.com/api/3.0/venues/62388/calendar.json?apikey=Y77nbW56nkfpIpw9");
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchema.json"));
+                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchemaAll.json"));
```

Comment:
Changed schema to SongkickSchemaAll so that the same schema could be used for all venues.